Jenkinsfile

```diff
@@ -183,13 +252,53 @@ stage('Unit Test') {
         }
       }
     }
-  }
+  },
+  'Python2/3: CPU Win':{
+    node('windows') {
+      ws('workspace/ut-python-cpu') {
+        init_git_win()
+        unstash 'vc14_cpu'
+        bat '''rmdir /s/q pkg_vc14_cpu
+7z x -y vc14_cpu.7z'''
+        bat """xcopy C:\\mxnet\\data data /E /I /Y
+xcopy C:\\mxnet\\model model /E /I /Y
+call activate py3
+set PYTHONPATH=${env.WORKSPACE}\\pkg_vc14_cpu\\python
+C:\\mxnet\\test_cpu.bat"""
+                        bat """xcopy C:\\mxnet\\data data /E /I /Y
+xcopy C:\\mxnet\\model model /E /I /Y
+call activate py2
+set PYTHONPATH=${env.WORKSPACE}\\pkg_vc14_cpu\\python
+C:\\mxnet\\test_cpu.bat"""
+      }
+     }
+   },
+   'Python2/3: GPU Win':{
+     node('windows') {
+       ws('workspace/ut-python-gpu') {
+         init_git_win()
+         unstash 'vc14_gpu'
+         bat '''rmdir /s/q pkg_vc14_gpu
+7z x -y vc14_gpu.7z'''
+         bat """xcopy C:\\mxnet\\data data /E /I /Y
```

Comment:
why we need to copy data and model? because the download program doesn't work on windows?